Makefile

```diff
@@ -19,18 +19,15 @@ LIB = libnsdl.a
 SRCS := \
 	source/libNsdl/src/sn_grs.c \
 	source/libNsdl/src/sn_nsdl.c \
-	source/libCoap/src/sn_coap_protocol.c \
-	source/libCoap/src/sn_coap_parser.c \
-	source/libCoap/src/sn_coap_header_check.c \
-	source/libCoap/src/sn_coap_builder.c \
 
 override CFLAGS += -DVERSION='"$(VERSION)"'
 
 override CFLAGS += -Isource/libNsdl/src/include/
-override CFLAGS += -Isource/libCoap/src/include/
 SERVLIB_DIR := ../libService
 override CFLAGS += -I$(SERVLIB_DIR)/libService
 override CFLAGS += -Insdl-c/
+override CFLAGS += -I../libcoap/libcoap/
+override CFLAGS += -I../libcoap/source/include/
```

Comment:
Do you really need to include /source/include? If you do, you're cheating a bit - as they should be private headers, not accessed from outside the library.

I guess you could argue that this component is coupled in some way. Would only cause a problem if you for some reason had binary coap library and were trying to compile source nsdl library.